packages/cfd/src/Containers/mt5-migration-modal/mt5-migration-account-icons.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { Jurisdiction, JURISDICTION_MARKET_TYPES } from '@deriv/shared/src/utils/constants';
```

Comment:
Just '@deriv/shared' would be enough. You do not need to give the exact path to it.